include/ck_tile/host/arg_parser.hpp

```diff
@@ -148,18 +197,21 @@ class ArgParser
 
     int get_int(const std::string& name) const
     {
+        assert(("The param contians invalid number", ValidNumber::isNumber(get_str(name)) == true));
         int value = atoi(input_map.at(name).value.c_str());
         return value;
     }
 
     uint32_t get_uint32(const std::string& name) const
     {
+        assert(("The param contians invalid number", ValidNumber::isNumber(get_str(name))));
         uint32_t value = strtoul(input_map.at(name).value.c_str(), nullptr, 10);
         return value;
     }
 
     uint64_t get_uint64(const std::string& name) const
     {
+        assert(("The param contians invalid number", ValidNumber::isNumber(get_str(name))));
```

Comment:
Let's not be so pedantic. This is redundant. And moreover the `assert` function is no-op in Release build so it won't be checked anyway.